modules/zero-trust-manager-federation-configuration.adoc

```diff
@@ -0,0 +1,237 @@
+= Configuring SPIRE federation with the https_spiffe profile
+
+[role="_abstract"]
+The {zero-trust-full} includes SPIRE Federation support, allowing multiple independent SPIRE deployments to establish trust relationships. This procedure demonstrates how to configure federation using the `https_spiffe` profile, which uses SPIFFE-based TLS authentication between SPIRE servers.
+
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed the {zero-trust-full} on all clusters that will participate in the federation.
+
+* You have `cluster-admin` privileges on all participating clusters.
+
+* You have network connectivity between the clusters you intend to federate.
+
+.Procedure
+
+. Configure the `SpireServer` custom resource on each cluster to enable federation with the `https_spiffe` profile. The `https_spiffe` profile uses SPIFFE-based TLS authentication, where SPIRE servers authenticate to each other using their own SVIDs (SPIFFE Verifiable Identity Documents).
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: SpireServer
+metadata:
+  name: cluster
+spec:
+  trustDomain: cluster1.example.com
+  federation:
+    bundleEndpoint:
+      profile: https_spiffe
+      refreshHint: 300
+    managedRoute: "true"
+----
+
+* The `trustDomain` field sets a unique trust domain for each cluster.
+
+* The `profile` field uses the `https_spiffe` profile for SPIFFE-based TLS authentication.
+
+* The `refreshHint` field suggests intervals (in seconds) for remote servers to refresh the trust bundle. Range: 60-3600 seconds.
+
+* The `managedRoute` field enables automatic route creation by the Operator.
+
+. Apply the configuration changes by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f spire-server.yaml
+----
+
+. Check the status of the SPIRE Server by entering the following command. Wait for the `Ready` status to be returned.
++
+[source,terminal]
+----
+$ oc get spireserver cluster -w
+----
+. Verify that the federation route has been created:
++
+[source,terminal]
+----
+$ oc get route -n zero-trust-workload-identity-manager | grep federation
+----
++
+.Example output
+[source,terminal]
+----
+NAME                      HOST/PORT                                    PATH   SERVICES        PORT    TERMINATION
+spire-server-federation   federation.apps.cluster1.example.com               spire-server     8443    passthrough
+----
+
+. Fetch the trust bundle from each remote cluster's federation endpoint:
++
+[source,terminal]
+----
+$ curl https://federation.apps.cluster2.example.com > cluster2-bundle.json
+----
++
+[NOTE]
+====
+For `https_spiffe` profile, you might need to use `-k` flag if the certificate is not trusted by your system's CA bundle:
+====
+[source,terminal]
+----
+$ curl -k https://federation.apps.cluster2.example.com > cluster2-bundle.json
+----
++
+The response contains the trust bundle in JSON Web Key Set (JWKS) format:
```

Comment:
This line needs fixed: 

<img width="868" height="498" alt="Screenshot From 2025-12-18 13-41-05" src="https://github.com/user-attachments/assets/ccf7f769-9446-43ea-a1cb-c7bb420ef998" />